### DISCUSSION TITLE
meshcentral  <=1.1.20 Severity: critical

### DISCUSSION THREAD

**Discussion by harmelgill** [Ylianst/MeshCentral#6105] [Issues]
Hi is this an issue on 

support@support:~$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.4 LTS"
support@support:~$

# npm audit report

meshcentral  <=1.1.20
Severity: critical
Ylianst MeshCentral Missing SSL Certificate Validation - https://github.com/advisories/GHSA-8xw6-9h78-c89j
MeshCentral algorithm-downgrade issue - https://github.com/advisories/GHSA-wpxw-5xfm-x22v
MeshCentral cross-site websocket hijacking (CSWSH) vulnerability - https://github.com/advisories/GHSA-cp68-qrhr-g9h8
fix available via `npm audit fix --force`
Will install meshcentral@1.1.22, which is outside the stated dependency range
node_modules/meshcentral

1 critical severity vulnerability

To address all issues, run:
  npm audit fix --force
support@support:~$

**Comment by si458** [comment]
it's recommended to always run the latest, so `npm install meshcentral`
